templates/publisher/listing.html

```diff
@@ -316,21 +316,9 @@
                   checked="checked"
                 {% endif %}
                 />
-              <label class="u-no-margin--top p-tooltip p-tooltip--top-left" for="public-metrics-enabled">Display public popularity charts
-                <span class="p-tooltip__message">These display relative popularity, not exact numbers.</span>
+              <label class="p-tooltip p-tooltip--top-left" for="public-metrics-enabled">Display public popularity charts
+                <span class="p-tooltip__message u-sv-1">These display relative popularity, not exact numbers.</span>
```

Comment:
This seems to be a bug with Vanilla tooltip (that there is more padding in the bottom). Or maybe it's intended. I wouldn't fix that with `u-sv-1`, as it's a hack.

Reply:
Agreed, I've removed this and the spacing utility and created https://github.com/vanilla-framework/vanilla-framework/issues/1882